includes/definitions/discovery/teldat.yaml

```diff
@@ -6,3 +6,42 @@ modules:
         hardware_template: '{{ TELDAT-MIB::telAdminStatusSystemBoardType.0 }} (rev {{ TELDAT-MIB::telAdminStatusSystemBoardRevision.0 }})'
         serial: TELDAT-MIB::telAdminStatusSystemNumSerie.0
         version: TELDAT-MIB::telAdminStatusSystemAppVersion.0
```

Comment:
Maybe You could add a line like
`sysDescr_regex: '/S\/N: (?<serial>\S+)/'`
To extract the serial number from your device, (need to regenerate json after that)

Reply:
Sure no worries. I thought of that... Let's take advantage, since we are here, let's do that too.
Thx

Reply:
more info:
On these TELDAT devices we have two OID's that, once put together, will get us the SN: 9999 / 99999  ( OID1 / OID2 ).
On some TELDAT devices we have this in one OID only.
Can we get this Regex used as last resort ?

Most Cases: 
Router model TV BASE VDSL2/ADSL WLAN 30 10 CPU MIPS32 S/N: 792/32964  :
.1.3.6.1.4.1.2007.1.2.1.4.0 = Gauge32: 32964
.1.3.6.1.4.1.2007.1.2.1.5.0 = STRING: "792"
.1.3.6.1.4.1.2007.1.2.1.12.0 ( ::= doesn't exists ! )

Some Devices:
.1.3.6.1.4.1.2007.1.2.1.4.0 = Gauge32: 20300343
.1.3.6.1.4.1.2007.1.2.1.5.0 = STRING: "RNNM"
.1.3.6.1.4.1.2007.1.2.1.12.0 = STRING: "RNNFDM020300343"

Reply:
Ok, ignore this last post.
I was checking my inventory, /cause have alot of devices +1k, and all of them have the "S/N:" string ! even in different models... so let's use this to our advantage.

should I remove this :  " serial: TELDAT-MIB::telAdminStatusSystemNumSerie.0 " ???

Sure, I'll add this:
sysDescr_regex: '/S\/N: (?<serial>\S+)/'

Reply:
yup.
I had to replace " serial: TELDAT-MIB::telAdminStatusSystemNumSerie.0 " . otherwise we on get the last part of the S/N.

Reply:
ok! commited. Tested discoveries all is well.

Reply:
Im pretty sure sysDescr_regex is used with lowest priority.
But I think we can do a bit of trickery to support your "else" statement... let me test

Reply:
hi. I think It's not needed, because I checked all devices on my inventory, and we can push the serial number from the 'S/N:' , I'm leaving these coments here for future memory, so that if any case arises, we have enough info do release a patch in the future. For now, I see no need for us to loose more time around this. It works well from the Description, and in fact, it's better than it was before.
Cheers.